src/test/java/it/gov/pagopa/payhub/activities/service/ingestionflow/organization/OrganizationProcessingServiceTest.java

```diff
@@ -0,0 +1,245 @@
+
+  @Mock
+  private Path workingDirectory;
+
+  @Mock
+  private OrganizationMapper mapperMock;
+
+  @Mock
+  private OrganizationService organizationServiceMock;
+
+  @Mock
+
+  private OrganizationProcessingService service;
+
+  @BeforeEach
+  void setUp() {
+    service = new OrganizationProcessingService(mapperMock, errorsArchiverServiceMock,
+        organizationServiceMock);
+  }
+
+  @AfterEach
+  void verifyNoMoreInteractions(){
+    Mockito.verifyNoMoreInteractions(
+        mapperMock,
+        organizationServiceMock);
+  }
+
+  @Test
+  void processOrganizationWithNoErrors() {
+    // Given
+    IngestionFlowFile ingestionFlowFile = buildIngestionFlowFile();
+    OrganizationIngestionFlowFileDTO dto = mock(OrganizationIngestionFlowFileDTO.class);
+    Mockito.when(dto.getBrokerCf()).thenReturn("brokerFC");
+
+    Organization orgFromService = Organization.builder()
+        .brokerId(1L)
+        .orgFiscalCode("brokerFC")
+        .ipaCode("ipaCode")
+        .orgName("orgName")
+        .status(OrganizationStatus.ACTIVE)
+        .flagNotifyIo(true)
+        .flagPaymentNotification(true)
+        .flagNotifyOutcomePush(true)
+        .build();
+
+    OrganizationRequestBody mappedOrg = OrganizationRequestBody.builder()
+        .ipaCode("ipaCode")
+        .orgFiscalCode("orgFiscalCode")
+        .orgName("orgName")
+        .status(OrganizationStatus.ACTIVE)
+        .flagNotifyIo(false)
+        .flagNotifyOutcomePush(false)
+        .flagPaymentNotification(false)
+        .build();
+
+    Organization createdOrg = Organization.builder()
+        .ipaCode("ipaCode1")
+        .orgFiscalCode("orgFiscalCode1")
+        .orgName("orgName1")
+        .status(OrganizationStatus.ACTIVE)
+        .flagNotifyIo(false)
+        .flagNotifyOutcomePush(false)
+        .flagPaymentNotification(false)
+        .build();
+
+    Mockito.when(organizationServiceMock.getOrganizationById(ingestionFlowFile.getOrganizationId()))
+        .thenReturn(Optional.of(orgFromService));
+    Mockito.when(mapperMock.map(dto, orgFromService.getBrokerId())).thenReturn(mappedOrg);
+    Mockito.when(organizationServiceMock.createOrganization(mappedOrg)).thenReturn(createdOrg);
+
+    // When
+    OrganizationIngestionFlowFileResult result = service.processOrganization(
+        Stream.of(dto).iterator(), List.of(),
+        ingestionFlowFile, workingDirectory);
+
+    // Then
+    Assertions.assertEquals(1L, result.getProcessedRows());
+    Assertions.assertEquals(1L, result.getTotalRows());
+    Assertions.assertNotNull(result.getOrganizationIpaCodeList());
+    Assertions.assertEquals(1, result.getOrganizationIpaCodeList().size());
+    Mockito.verify(organizationServiceMock).getOrganizationById(ingestionFlowFile.getOrganizationId());
+    Mockito.verify(mapperMock).map(dto, orgFromService.getBrokerId());
+    Mockito.verify(organizationServiceMock).createOrganization(mappedOrg);
+    Mockito.verifyNoInteractions(errorsArchiverServiceMock);
```

Comment:
let's add this mock on Mockito noMoreInvocation configuration on @AfterEach method, thus you could remove this line